exercises/sum-of-multiples/.meta/generator/sum_of_multiples_case.rb

```diff
@@ -0,0 +1,11 @@
+require 'generator/exercise_case'
+
+class SumOfMultiplesCase < Generator::ExerciseCase
+  using Generator::Underscore
+
+  def workload
+    assert_expected = "assert_equal #{expected.underscore}"
+    value = "SumOfMultiples.new(#{factors.join(', ')}).to(#{limit})"
+    indent_lines(["#{assert_expected}, #{value}"], 4)
```

Comment:
Feedback rather than a change request:

The grouping here is logically confusing as in `assert_expected` you have only 2/3 of a valid `assert_equal` call. 
`value` is good.
constructing the string at the end is ok due to your variable naming, but confusing because `assert_expected` looks like a minitest method but it isn't one.


I would have done this:
```
result = "SumOfMultiples.new(#{factors.join(', ')}).to(#{limit})"
indent_lines(["assert_equal #{expected.underscore}, #{result}"], 4)
```